benchmark/scenarios/test/test_write_request_load.py

```diff
@@ -472,7 +488,11 @@ def test_scenario_stops_only_when_no_outstanding_requests(self):
         c.advance(delay)
         self.successResultOf(d)
 
-    def test_scenario_timeouts_if_requests_not_completed(self):
+        # Suppress Eliot error for unflushed exceptions
+        logger.flushTracebacks(FakeNetworkError)
```

Comment:
I *think* this should be moved to after the `fail_requests = False` statement, since AFAICT, after then, everything ought to be working, and any logged exceptions after then will be genuine errors.

However, the whole thing smells a bit like a bug to me. Since we're creating the exceptions, we should be able to add errbacks that suppress the logging of tracebacks. Would welcome @itamarst 's opinion.